classes/simplemap.php

```diff
@@ -2289,27 +2289,27 @@ function show_toolbar( $title = '' ) {
 			<table class="sm-toolbar" cellspacing="0" cellpadding="0" border="0">
 				<tr>
 					<td class="sm-page-title">
-						<h2><?php _e( $title, 'simplemap' ); ?></h2>
+						<h2><?php _e( $title, 'SimpleMap' ); ?></h2>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->